qiskit/scheduler/lowering.py

```diff
@@ -73,6 +75,7 @@ def get_measure_schedule(qubit_mem_slots: Dict[int, int]) -> CircuitPulseDef:
             for qubit in qubits:
                 try:
                     meas_q = circuit.calibrations[Measure().name][((qubit,), params)]
+                    meas_q = base_qobj_transform(meas_q)
```

Comment:
Why are these two additions required?

Reply:
Because this `meas_q` schedule is filtered by acquire channel (if measure schedule is built with the builder and attached to circuit, the lowering crashes). And we don't have any pass that convert attached calibration into schedule. We can write transpiler pass that runs `base_qobj_transform` to calibrations instead.